datalad/distribution/tests/test_uninstall.py

```diff
@@ -402,8 +412,21 @@ def test_kill(path):
     res = ds.remove(on_failure='ignore')
     assert_result_count(
         res, 1,
-        status='error', path=testfile,
-        message='configured minimum number of copies not found')
+        status='error', path=testfile)
+    # Following two assertions on message are relying on the actual error.
+    # We have a second result with status 'impossible' for the ds, that we need
+    # to filter out for those assertions:
+    err_result = [r for r in res if r['status'] == 'error'][0]
+    assert_result_values_cond(
+            [err_result], 'message',
+            lambda x: "(Use --force to override this check, "
+                      "or adjust numcopies.)" in x
+    )
+    assert_result_values_cond(
+        [err_result], 'message',
+        lambda x: "configured minimum number of copies not found" in x or
+        "Could only verify the existence of 0 out of 1 necessary copies" in x
+    )
```

Comment:
seems to be a repeating pattern.  Shouldn't we have a dedicated helper
`assert_result_not_enough_copies(err_result, 'message')`

Reply:
That's quite a specific helper. Actually, I think we should unify on lower level, so the call to drop would raise an `InsufficientNumCopiesError`, that could be detected independently on a version specific output. Then the result can have a stable message, that we can test for herein as before.